src/main/java/io/toast/tk/adapter/web/AbstractWebActionAdapter.java

```diff
@@ -148,4 +151,20 @@ public ITestResult closeBrowser() {
 		driver.getWebDriver().quit();
 		return new SuccessResult();
 	}
+	
+	@Action(id="double_click_on_web_component", action = "Double click on " + WEB_COMPONENT, description = "")
+	public ITestResult doubleClickOn(IWebAutoElement<WebElement> pageField) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS00112)